contracts/optimistic-ethereum/OVM/bridge/OVM_MultiRelay.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >0.5.0 <0.8.0;
+pragma experimental ABIEncoderV2;
+
+/* Interface Imports */
+import { iOVM_L1CrossDomainMessenger } from "../../iOVM/bridge/iOVM_L1CrossDomainMessenger.sol";
+
+/* Contract Imports */
+import { Ownable } from "../../libraries/resolver/Lib_Ownable.sol";
+
+contract OVM_MultiRelay is Ownable {
+
+    /**********
+     * Events *
+     **********/
+
+    event RelaySet(address _newAddress);
+
+    struct L2ToL1Message {
+        address target;
+        address sender;
+        bytes message;
+        uint256 messageNonce;
+        iOVM_L1CrossDomainMessenger.L2MessageInclusionProof proof;
+    }
+
+    address public relay;
```

Comment:
I think this could be a `iOVM_L1CrossDomainMessenger` instead of just an `address`

Reply:
I agree that'd be better.